Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -103,9 +122,15 @@ final class ExperimentalAnimationLayer: CALayer {
 
 extension ExperimentalAnimationLayer: RootAnimationLayer {
 
-  var currentFrame: CGFloat {
-    get { 0 }
-    set { /* Currently unsupported */ }
+  var currentFrame: AnimationFrameTime {
+    get {
+      0 // TODO: how do we retrieve the realtime animation progress?
```

Comment:
I assume `presentation().timeOffset` doesn't work here?

Reply:
I tried that and it was always `0`

Reply:
It seems like the idiomatic way to track realtime animation progress with Core Animation is to create a placeholder `CABasicAnimation` that animates a `"progress"` key or something like that ([discussion on stack overflow](https://stackoverflow.com/questions/18827973/core-animation-progress-callback/19024971#19024971), which is basically exactly how this works in the existing animation engine)